package.json

```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "name": "cerestats",
-  "version": "0.31.0",
+  "version": "0.32.0",
```

Comment:
Do we need to change the version in `./backend/package.json`?

Reply:
Yes, we do. Thanks. 
https://github.com/Cerebellum-Network/cerestats/pull/156